src/Npgsql/Internal/TypeHandlers/FullTextSearchHandlers/TsVectorHandler.cs

```diff
@@ -74,7 +74,7 @@ public override async ValueTask<NpgsqlTsVector> Read(NpgsqlReadBuffer buf, int l
         // TODO: Implement length cache
         /// <inheritdoc />
         public override int ValidateAndGetLength(NpgsqlTsVector value, ref NpgsqlLengthCache? lengthCache, NpgsqlParameter? parameter)
-            => 4 + value.Sum(l => Encoding.UTF8.GetByteCount(l.Text) + 1 + 2 + l.Count * 2);
+            => 4 + value.Sum(l => Encoding.UTF8.GetByteCount((string)l.Text) + 1 + 2 + l.Count * 2);
```

Comment:
But why?

Reply:
No idea how that got in... good catch!

Reply:
You even made me double-check by fetching the branch 🤣